src/views/EditSidebar.vue

```diff
@@ -93,6 +93,9 @@
 			<ActionButton v-if="canDelete && canCreateRecurrenceException" icon="icon-delete" @click="deleteAndLeave(true)">
 				{{ $t('calendar', 'Delete this and all future') }}
 			</ActionButton>
+			<ActionButton v-if="!isNew" icon="icon-file" @click="duplicateEvent()">
```

Comment:
I'd say *delete* should always be the last item in the list. Move this new action up one position